packages/renderer/src/stores/dashboard/dashboard-page-registry.svelte.ts

```diff
@@ -0,0 +1,102 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ListOrganizerItem } from '@podman-desktop/ui-svelte';
+import type { Component } from 'svelte';
+
+import { createExploreFeatures } from './dashboard-page-registry-explore-features';
+import { createExtensionBanners } from './dashboard-page-registry-extension-banners.svelte';
+import { createLearningCenter } from './dashboard-page-registry-learning-center.svelte';
+import { createProviders } from './dashboard-page-registry-providers.svelte';
+import { createReleaseNotesBox } from './dashboard-page-registry-release-notes.svelte';
+
+export interface DashboardPageRegistryEntry {
+  id: string;
+  hidden?: boolean;
+  originalOrder: number;
+  component?: Component;
+}
+
+export const dashboardPageRegistry = $state<{ entries: DashboardPageRegistryEntry[] }>({ entries: [] });
+
+let values: DashboardPageRegistryEntry[] = [];
+
+function setupDashboardPageRegistry(val: DashboardPageRegistryEntry[]): void {
+  val.push(createReleaseNotesBox());
+  val.push(createExtensionBanners());
+  val.push(createExploreFeatures());
+  val.push(createLearningCenter());
+  val.push(createProviders());
+}
+
+const init = (): void => {
+  setupDashboardPageRegistry(values);
+  // Set originalOrder for each entry based on their position
+  values.forEach((entry, index) => {
+    entry.originalOrder = index;
+  });
+
+  // Update the state
+  dashboardPageRegistry.entries = values;
+};
+
+init();
+
+export const resetDashboardPageRegistries = async (): Promise<void> => {
+  // Reset to initial state
+  values = [];
```

Comment:
suggestion: if you apply https://github.com/podman-desktop/podman-desktop/pull/14439/files#r2450982803, you can remove this line because init will replace values, so no need to empty it